field-library/canonicalLinks.json

```diff
@@ -0,0 +1,218 @@
+[
+  {
+    "name": "Canonical Links",
```

Comment:
Should we rename this to _Podcast Links_ or _Media Links_ or something else more descriptive? Not apparent that a “Canonical Links” field would contain Spotify / YouTube / Anchor.fm etc options.